ovos_config/mycroft.conf

```diff
@@ -607,10 +607,20 @@
     "module": "ovos-tts-plugin-server",
     "fallback_module": "ovos-tts-plugin-mimic3-server",
     "ovos-tts-plugin-mimic": {
-        "voice": "ap"
+        "voice": "ap",
+        // persistent cache
+        "persist_cache": false,
+        "preloaded_cache": "",
+        "min_free_percent": 75,
+        "persist_thresh": 1
```

Comment:
docstr saying this is how many times a utterance must be seen in order to be added to persistent cache